packages/gdb.rb

```diff
@@ -13,12 +13,22 @@ class Gdb < Autotools
   source_sha256 'bcfcd095528a987917acf9fff3f1672181694926cc18d609c99d0042c00224c5'
   binary_compression 'tar.zst'
 
-  binary_sha256({
-    aarch64: '877bea13576e5b36931d69624bef3471b69c4637b10ede10ac34cf79164f6167',
-     armv7l: '877bea13576e5b36931d69624bef3471b69c4637b10ede10ac34cf79164f6167',
-       i686: '432295e34df829235f1a66e8c7573a6bfa4e50261e1f9d0c98bcaea9836c7206',
-     x86_64: 'ccc89a610cdfff8e227dec2c1557c5c896dd6ca1878e29c8c148cf6371d8c611'
-  })
+  case CREW_GCC_VER
+  when 'gcc14'
```

Comment:
What happens if gcc < 14?

Reply:
We don't build gdb for installations with older gcc, so that would be incompatible.